app/master/subjob.py

```diff
@@ -103,29 +98,6 @@ def atomic_commands(self):
         job_command = self.job_config.command
         return ['{} {}'.format(atom.command_string, job_command) for atom in self._atoms]
 
-    def get_atom_artifact_name(self, atom_id):
```

Comment:
How you think about keeping some of the helper methods of subjob but call the static method as implementation? Keeping the code DRY is awesome. But calling static methods all the time does seem to be very OO.